pkg/splunk/enterprise/indexercluster.go

```diff
@@ -67,8 +67,18 @@ func ApplyIndexerCluster(client splcommon.ControllerClient, cr *enterprisev1.Ind
 		}
 	}()
 
+	// create or update general config resources
```

Comment:
Any reason we moved this up? Same comment for ClusterMaster.go

Reply:
Same, what we did with other CRs, creation of secrets before we call ApplyMonitoringConsole() during deletion, for UTs